cmd/fluxctl/realease_command_test.go

```diff
@@ -0,0 +1,210 @@
+package main //+integration
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/gorilla/mux"
+	"github.com/weaveworks/flux"
+	transport "github.com/weaveworks/flux/http"
+	"github.com/weaveworks/flux/jobs"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+	"testing"
+)
+
+func TestReleaseCommand_CLIConversion(t *testing.T) {
+	for _, v := range []struct {
+		args           []string
+		expectedParams map[string]string
+	}{
+		{[]string{"--update-all-images", "--all"}, map[string]string{
+			"service": string(flux.ServiceSpecAll),
+			"image":   string(flux.ImageSpecLatest),
+			"kind":    string(flux.ReleaseKindExecute),
+		}},
+		{[]string{"--update-all-images", "--all", "--dry-run"}, map[string]string{
+			"service": string(flux.ServiceSpecAll),
+			"image":   string(flux.ImageSpecLatest),
+			"kind":    string(flux.ReleaseKindPlan),
+		}},
+		{[]string{"--no-update", "--all"}, map[string]string{
+			"service": string(flux.ServiceSpecAll),
+			"image":   string(flux.ImageSpecNone),
+			"kind":    string(flux.ReleaseKindExecute),
+		}},
+		{[]string{"--update-image=alpine:latest", "--all"}, map[string]string{
+			"service": string(flux.ServiceSpecAll),
+			"image":   "alpine:latest",
+			"kind":    string(flux.ReleaseKindExecute),
+		}},
+		{[]string{"--update-all-images", "--service=default/flux"}, map[string]string{
+			"service": "default/flux",
+			"image":   string(flux.ImageSpecLatest),
+			"kind":    string(flux.ReleaseKindExecute),
+		}},
+		{[]string{"--update-all-images", "--all", "--exclude=default/test,default/yeah"}, map[string]string{
```

Comment:
Do we expect comma-separated values or multiple options like `--exclude foo --exclude bar`? (or both)